### ISSUE TITLE
Gallery of images to events

### ISSUE THREAD

**Issue by viniciusalonso** [guru-pr/gurupr#21]
Each event with a images gallery, for better disclose and show how to the are events for those never were

**Comment by vyper** [comment]
Yes, this is a very important... maybe us can this make together with #8.

**Comment by brunoocasali** [comment]
The website has a own cloud server, which image server is chosen for this ? AWS/Cloudinary/Dropbox ?

**Comment by vyper** [comment]
My choice is "better free"! (:

**Comment by neves** [comment]
+1 for Cloudinary. The free plan has more than enough space and has plugin for Rails.

**Comment by rodrigobaron** [comment]
I can help implements that in this weekend

**Comment by nandomoreirame** [comment]
I can help with the Frontend this 🤘🏻

**Comment by vyper** [comment]
I think that a good idea add all pictures of event in the event page.

# Ideas...

![screen shot 2016-06-09 at 17 30 28](https://cloud.githubusercontent.com/assets/22549/15945379/978b2016-2e68-11e6-9d15-4e87e1c18f51.png)

**Comment by vyper** [comment]
Other ideas? (:

**Comment by rodrigobaron** [comment]
nice, 
+1 for **option 1** without carousel. 
Also in front page will have some preview or something like?

**Comment by vyper** [comment]
In homepage I think we need other feature. We need a option to choice which information will be expose there. Maybe an event or event information (pictures) or a news (news not exist yet). 😁

**Comment by adilsoncarvalho** [comment]
And where are those images coming from? Some options I'd consider to store and provide storage:

- As they're event based what about using [Instagram](https://www.instagram.com/developer/endpoints/tags/) to store the images using tags to filter? It could be handy to present any picture with that hashtag (of course this idea has some caveats)

- Other option could be [Google Photos](https://developers.google.com/picasa-web/) as it offers unlimited storage for pictures on high resolution and a nice API to fetch the pictures.

- At last there is [Flickr's 1TB storage](https://www.flickr.com/services/api/) that could be used as well.

What do you think, guys?

**Comment by vyper** [comment]
I read about "Google Photos"... but [see this](http://stackoverflow.com/a/32525894/1165373).
